### PR TITLE
Add the README for the templates

### PR DESCRIPTION
**Pull request by chelseatroy**
### Goal(s) of this Pull Request:

- Transfer the README that is under active review in moz-mlops, so we have just one repo under active development with our ops tools. See [that review ](https://github.com/mozilla/moz-mlops/pull/4) for context on changes being made to this file. Moving forward, the file will live here, but first it's being edited _there_. 

### Acceptance criteria:

In order to approve, reviewer must be able to...

- [ ] Use the instructions listed in this file to be able to run the `template_flow.py` flow successfully.

### CODE REVIEW

File: src/mozmlops/templates/template_flow.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from mozmlops.artifact_store import ArtifactStore
```
Comment (Dexterp37) [inline src/mozmlops/templates/template_flow.py:14]: This won't work unless `mozmlops` is installed: I'd add a one line comment above this line to mention it. It shouldn't be a big deal, this is going to change once the package gets published and I doubt anyone is going to use this in the next week or so.

File: src/mozmlops/templates/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Training and Deploying Models in The Cloud
```
Comment (Dexterp37) [inline src/mozmlops/templates/README.md:1]: Are you planning on amending this file as part of this PR?

Comment (Dexterp37) [inline src/mozmlops/templates/README.md:1]: Please add a reference to templates (and kindly point to this doc!) in the top level README.

**Review by Dexterp37 — CHANGES_REQUESTED** [review]

File: src/mozmlops/templates/template_flow.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from artifact_store import ArtifactStore
```
[the remark this answers is not in the document]
Comment (chelseatroy) [reply src/mozmlops/templates/template_flow.py]: It does not. I was working on both branches at the time and committed this change to the wrong one by mistake. Sorry about that! I have changed it in the branch it was supposed to be in and merged it to main.

File: src/mozmlops/templates/template_flow.py (hunk 1 above)
Comment (chelseatroy) [reply src/mozmlops/templates/template_flow.py:14]: Can do!

File: src/mozmlops/templates/template_flow.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dexterp37) [reply src/mozmlops/templates/template_flow.py]: Whoops, this was actually a stray comment that should not have been submitted 😵‍💫

File: src/mozmlops/templates/README.md (hunk 2 above)
Comment (chelseatroy) [reply src/mozmlops/templates/README.md:1]: Done!

**Review by Dexterp37 — APPROVED** [review]

**Pull request merged**